mongodb-monitoring/templates/_deployment.tpl

```diff
@@ -0,0 +1,76 @@
+{{ define "mongodb_exporter_deployment" }}
```

Comment:
this file is a bit weird. you copied it from that other one?
I don't think it will work like this.

Reply:
yes copied from the upstream